src/compiler/watchPublic.ts

```diff
@@ -706,6 +712,10 @@ namespace ts {
             }
         }
 
+        function watchExtendedConfigFile(extendedConfigFile: string) {
+            return watchFile(extendedConfigFile, scheduleProgramReload, PollingInterval.High, watchOptions, WatchType.ConfigFile);
+        }
+

```

Comment:
Similar changes are needed in tsserver and test for configured project. we would probably want common map between projects for extended source files so they share the watch and schedule update for all the projects that extend the source files

Reply:
This comment was not addressed by the latest commit (4f24e47). I'll do some investigating to make the proposed changes to tsserver and will update with more commits. Any pointers to relevant files/sections would be appreciated :)